frontend/src/metabase/modes/components/drill/ColumnFilterDrill.jsx

```diff
@@ -1,18 +1,22 @@
 /* eslint-disable react/prop-types */
 import React from "react";
 import { t } from "ttag";
+import _ from "underscore";
+import { TYPE, isa } from "metabase/lib/types";
 
 import Filter from "metabase-lib/lib/queries/structured/Filter";
-
 import FilterPopover from "metabase/query_builder/components/filters/FilterPopover";
 
+const INVALID_TYPES = [TYPE.Structured];
+
 export default function ColumnFilterDrill({ question, clicked }) {
   const query = question.query();
   if (
     !question.isStructured() ||
     !query.isEditable() ||
     !clicked ||
     !clicked.column ||
+    _.any(INVALID_TYPES, type => isa(clicked.column.base_type, type)) ||
```

Comment:
Should we favor the native Array's `.some`? There are currently like 100 `.some` and 24 `_.any` from a text search. So `.some` seems more established.

Reply:
I like this. will do. 👍